features/gezag-meerderjarige/meerderjarige-rni.feature

```diff
@@ -0,0 +1,166 @@
+      | naam        | waarde                                                               |
+      | type        | GezagNietTeBepalen                                                   |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet in Nederland woont. |
+
+      Voorbeelden:
+      | geboortedatum            | leeftijd      | omschrijving                                                                             |
+      | deze maand - 5 jaar      | 4 of 5 jaar   | met alleen maand en jaar van geboortedatum bekend en is de hele maand jonger dan 18 jaar |
+      | dit jaar - 5 jaar        | 4 of 5 jaar   | met alleen jaar van geboortedatum bekend en is het hele jaar jonger dan 18 jaar          |
+      | volgende maand - 18 jaar | 17 jaar       | met alleen maand en jaar van geboortedatum bekend en wordt volgende maand 18 jaar        |
+      | dit jaar - 17 jaar       | 16 of 17 jaar | met alleen jaar van geboortedatum bekend en is of wordt dit jaar 17 jaar                 |
+
+    Abstract Scenario: persoon in RNI <omschrijving>
+      Gegeven de persoon 'Mees' met burgerservicenummer '000000036'
+      * is ingeschreven in de RNI
+      * heeft de volgende gegevens
+      | geboortedatum (03.10) |
+      | <geboortedatum>       |
+      * is in Nederland geboren
+      * is niet geëmigreerd geweest
+      * heeft 'Marlies' als ouder 1
+      * heeft 'Mark' als ouder 2
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+      Voorbeelden:
+      | geboortedatum          | leeftijd      | omschrijving                                                                            |
+      | deze maand - 35 jaar   | 34 of 35 jaar | met alleen maand en jaar van geboortedatum bekend en is de hele maand ouder dan 18 jaar |
+      | dit jaar - 35 jaar     | 34 of 35 jaar | met alleen jaar van geboortedatum bekend en is het hele jaar ouder dan 18 jaar          |
+      | deze maand - 18 jaar   | 17 of 18 jaar | met alleen maand en jaar van geboortedatum bekend en is of wordt deze maand 18 jaar     |
+      | vorige maand - 18 jaar | 18 jaar       | met alleen maand en jaar van geboortedatum bekend en werd vorige maand 18 jaar          |
+      | dit jaar - 18 jaar     | 17 of 18 jaar | met alleen jaar van geboortedatum bekend en is of wordt dit jaar 18 jaar                |
+      | dit jaar - 19 jaar     | 18 of 19 jaar | met alleen jaar van geboortedatum bekend en is 18 en is of wordt dit jaar 19 jaar       |
+      
+
+  Regel: een persoon die in RNI staat ingeschreven met een volledig onbekende geboortedatum krijgt geen gezag geleverd
+    # wanneer de geboortedatum is gelijk aan 00000000, is de persoon meerderjarig 
```

Comment:
Dit klopt niet meer. Een onbekende geboortedatum leidt nu tot GezagNietTeBepalen. De pipeline faalt ook hierdoor.

Reply:
Ik zag 1 feature de faalde en dat was:
features/gezag-minderjarige/1.2-is-persoon-minderjarig-en-niet-overleden.feature
Deze heb ik zojuist gefixed.

Reply:
Top!